lib/packwerk.rb

```diff
@@ -81,6 +80,11 @@ module Checkers
       extend ActiveSupport::Autoload
 
       autoload :Checker
+      module Checker
```

Comment:
This does work the way I want, since we are opening up the `Checker` module and setting up an autoload. This is similar to the same way `ReferenceChecker` is autoloaded, with `lib/packwerk.rb` above this line, however it's different in that we don't autoload `ReferenceChecking`, but not sure if it makes a difference:
```
  module ReferenceChecking
    extend ActiveSupport::Autoload

    autoload :ReferenceChecker
```

I think this is a style choice mostly and either way is fine with me – I've moved it to `checker.rb`!